packages/compass-components/src/components/interactive-popover.tsx

```diff
@@ -1,12 +1,37 @@
 import React, { useCallback, useEffect, useRef } from 'react';
-import { css } from '@leafygreen-ui/emotion';
+import { css, cx } from '@leafygreen-ui/emotion';
 import FocusTrap from 'focus-trap-react';
 
 import { Popover } from './leafygreen';
+import { spacing } from '@leafygreen-ui/tokens';
+import { uiColors } from '@leafygreen-ui/palette';
+import { transparentize } from 'polished';
+import { useTheme, Theme } from '../hooks/use-theme';
+import { gray8 } from '../compass-ui-colors';
+
+const borderRadius = spacing[2];
 
 const contentContainerStyles = css({
   display: 'flex',
   height: '100%',
+  alignItems: 'center',
+  borderRadius: borderRadius,
+  boxShadow: `0px 2px 4px -1px ${transparentize(0.85, uiColors.black)}`,
```

Comment:
Suggestion/though - Do we need to add an extra npm package for changing the color here? Seems like something we can avoid if we aren't going to be doing this color changing often, I think fewer dependencies usually makes things easier in the long run. 
Are these styles from somewhere in particular? Totally good to merge as is I think too.

Reply:
Buh not sure, that is what LG does internally to keep shadows consistent, I copy-pasted it. I can convert what's there to the shadow in the rebranded `uiColors.black`, but we would have that dependency in any case from leafygreen itself. I wish there would have been a way to just `import {boxShadow1, borderColor1} from 'leafygreen-constants'` or something, just to have some help keeping things 1-1 with the component library.

Btw we have something with a similar style for the "find in page" that we probably should align / color / border wise there too: 

<img width="275" alt="Screenshot 2022-08-31 at 15 11 53" src="https://user-images.githubusercontent.com/334881/187686404-acdc0ce8-7783-4e2f-af77-9a0a212f75c9.png">

Reply:
Sounds good, I think it's fine for now. We can add a `floatingBoxBoxShadowStyles` to `compass-components` that they both then use if we want.